src/Response/Exception/InvalidZipResponseException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
```

Comment:
Just to be sure: Have you run the `composer fix:php-cs` script?

Reply:
I did run `composer ci` which returned successfully (status `0`). I just noticed, that `fix:php-cs` isn't included in the *ci* tasks. Running it against c273b67 produces no changes, errors, or warnings.

Reply:
Those are different tasks: `composer ci` checks things, while `composer fix:php-cs` will modify things.